pom.xml

```diff
@@ -54,7 +55,7 @@
 	</repositories>
 
 	<properties>
-		<hapi-fhir.version>1.4-SNAPSHOT</hapi-fhir.version>
+		<hapi-fhir.version>2.4.0-PR660</hapi-fhir.version>
```

Comment:
I think that you're not using this variable anymore?

Reply:
Removed.